<details><summary>Failed to submit graph for 9126ce48cd6de7d6e4a22698b6ef4dd291b19ff8
</summary>

Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmppktggsp1/pytest-doctestplus-feedstock.yaml --custom-metadata-file /tmp/tmppktggsp1/custom_metadata.yaml --release-name pytest-doctestplus-feedstock-15-9126ce48cd6de7d6e4a22698b6ef4dd291b19ff8 pytest-doctestplus-feedstock/recipe', with error: 'INFO:root:Found feedstock pytest-doctestplus-feedstock at /tmp/tmppktggsp1/pytest-doctestplus-feedstock/recipe
INFO:root:Build/Test graph from pytest-doctestplus-feedstock
Created by sbx_user1051 on 2026-08-18 14:01:14
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmppktggsp1/pytest-doctestplus-feedstock.yaml --custom-metadata-file /tmp/tmppktggsp1/custom_metadata.yaml --release-name pytest-doctestplus-feedstock-15-9126ce48cd6de7d6e4a22698b6ef4dd291b19ff8 pytest-doctestplus-feedstock/recipe

INFO:root:Creating graph from feedstock at: pytest-doctestplus-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.12 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 4 unchanging vars:(467), changing vars (4): ['blas_impl', 'cuda_compiler_version', 'numpy', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_PKGS_DIRS = /tmp/conda_pkgs_z93vehf0.
WARNING: Number of parsed outputs does not match detected raw metadata blocks. Identified output block may be wrong! If you are using Jinja conditionals to include or exclude outputs, consider using `skip: true  # [condition]` instead.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_PKGS_DIRS
INFO:pbpcli.utils.expand_variants:Elapsed time for api.render: 39.70 seconds
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-aarch64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.11 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 3 unchanging vars:(467), changing vars (3): ['cuda_compiler_version', 'numpy', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_OVERRIDE_GLIBC = 2.28.
INFO:pbpcli.utils.common:Setting env var: CONDA_PKGS_DIRS = /tmp/conda_pkgs_kf9a144m.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_OVERRIDE_GLIBC
INFO:pbpcli.utils.common:Unsetting env var: CONDA_PKGS_DIRS
INFO:pbpcli.utils.expand_variants:Elapsed time for api.render: 30.11 seconds
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = osx-arm64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.11 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 2 unchanging vars:(470), changing vars (2): ['numpy', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_OVERRIDE_OSX = 12.1.
INFO:pbpcli.utils.common:Setting env var: CONDA_PKGS_DIRS = /tmp/conda_pkgs_ae_20gwe.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_OVERRIDE_OSX
INFO:pbpcli.utils.common:Unsetting env var: CONDA_PKGS_DIRS
INFO:pbpcli.utils.expand_variants:Elapsed time for api.render: 27.40 seconds
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = win-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.11 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 4 unchanging vars:(472), changing vars (4): ['blas_impl', 'cuda_compiler_version', 'numpy', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_OVERRIDE_WIN = 10.0.17763.
INFO:pbpcli.utils.common:Setting env var: CONDA_PKGS_DIRS = /tmp/conda_pkgs_dostg33e.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_OVERRIDE_WIN
INFO:pbpcli.utils.common:Unsetting env var: CONDA_PKGS_DIRS
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2092, in _get_contents
    template = env.get_or_select_template(filename)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 1016, in get_template
    return self._load_template(name, globals)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 771, in compile
    self.handle_exception(source=source_hint)
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/tmp/tmppktggsp1/pytest-doctestplus-feedstock/recipe/meta.yaml", line 38, in template
    {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_generate_diff_basic %} # [win]
    ^^^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got '='

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1524, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1445, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1912, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1912, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1308, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 877, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 323, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 603, in from_feedstock
    build_configs, test_configs = expand_build_matrix(
                                  ~~~~~~~~~~~~~~~~~~~^
        feedstock.recipe_dir_path, src_channels, parsed_user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 99, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels, user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 672, in variants_for_feedstock_and_platform
    metadata_tuples = api.render(
        parsed.recipe,
    ...<2 lines>...
        variants=user_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/api.py", line 60, in render
    metadata_tuples = render_recipe(
        recipe_path,
    ...<4 lines>...
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 1048, in render_recipe
    m = MetaData(str(recipe), config=config)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1250, in __init__
    self.parse_again(permit_undefined_jinja=True, allow_no_other_outputs=True)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1339, in parse_again
    self._get_contents(
    ~~~~~~~~~~~~~~~~~~^
        permit_undefined_jinja,
        ^^^^^^^^^^^^^^^^^^^^^^^
        allow_no_other_outputs=allow_no_other_outputs,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ),
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2107, in _get_contents
    raise CondaBuildUserError(
        f"Failed to render jinja template in {self.meta_path}:\n{str(ex)}"
    )
conda_build.exceptions.CondaBuildUserError: Failed to render jinja template in /tmp/tmppktggsp1/pytest-doctestplus-feedstock/recipe/meta.yaml:
expected token 'end of statement block', got '='
'
aws lambda request id: 22874157-17f8-510d-ba1c-23a97025392c
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%6022874157-17f8-510d-ba1c-23a97025392c%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)

</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,2 +1,0 @@
-build_env_vars:
-  ANACONDA_ROCKET_ENABLE_PY314 : yes

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,17 +1,17 @@
 {% set name = "pytest-doctestplus" %}
-{% set version = "1.6.0" %}
+{% set version = "1.7.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: f893d68370d19b418d58da0047d36a4feedac7ed28bc236858f484e994f1ac66
+  sha256: 64ae16dbda66c3db775ef596828d8d7adda09c6f34dd85099c119e4ff8cfe5b6
 
 build:
   number: 0
-  skip: true  # [py<39]
+  skip: true  # [py<310]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
@@ -23,17 +23,25 @@ requirements:
     - wheel
   run:
     - python
-    - pytest >=4.6
+    - pytest >=7.0
     - packaging >=17.0
 
-{% set deselect_tests = "" %}
-{% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_all" %}
+{% set deselect_tests = " --deselect=tests/test_doctestplus.py::test_remote_data_all" %}
 {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_url" %}
 {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_float_cmp" %}
 {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_ignore_whitespace" %}
 {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_ellipsis" %}
 {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_requires" %}
 {% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_remote_data_ignore_warnings" %}
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_encoding.py::test_legacy_diff" %}  # [win]
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_encoding.py::test_basic_file_encoding_diff" %}  # [win]
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_generate_diff_basic" %}  # [win]
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_generate_diff_multiline" %}  # [win]
+
+# AttributeError, broken by https://github.com/python/cpython/pull/115440
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_doctestplus.py::test_ufunc" %}  # [py==311]
+
+
 
 test:
   source_files:
@@ -42,9 +50,12 @@ test:
     - pytest_doctestplus
   requires:
     - pip
-    - pytest
     - setuptools >=30.3.0
+    - pytest-remotedata >=0.3.2
     - numpy
+    - sphinx
+    - pytest >=7.0
+    - git
   commands:
     - pip check
     - pytest -vv tests {{ deselect_tests }}

